app/src/main/java/org/wikipedia/page/PageTitle.java

```diff
@@ -150,15 +150,20 @@ public PageTitle(@Nullable String text, @NonNull WikiSite wiki, @Nullable String
             if (Arrays.asList(Locale.getISOLanguages()).contains(namespaceOrLanguage)) {
                 this.namespace = null;
                 this.wiki = new WikiSite(wiki.authority(), namespaceOrLanguage);
-            } else {
+                this.text = TextUtils.join(":", Arrays.copyOfRange(parts, 1, parts.length));
+            } else if (parts[1].length() > 0 && !Character.isWhitespace(parts[1].charAt(0)) && (parts[1].charAt(0) != '_')) {
```

Comment:
Really minor but the parentheses of  `(parts[1].charAt(0) != '_')` can be removed.

Reply:
done